vite.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from "vite";
+import reactRefresh from "@vitejs/plugin-react";
+import svgrPlugin from "vite-plugin-svgr";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  // This changes the out put dir from dist to build
+  // comment this out if that isn't relevant for your project
+  build: {
+    outDir: "build",
+  },
+  server: {
```

Comment:
Cool thing I found, if we add `open: true` to this server object I think the browser will automatically open like before if we're interested

Reply:
Sure I can implement that